packages/embedded-wallet-service-wagmi/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./interfaces/connector";
```

Comment:
general best practice is to manually specify the functions and types we want to export to prevent accidentally leaking internal API